### DISCUSSION TITLE
Change "atop" to "above"

### DISCUSSION THREAD

**Discussion by aidakdev** [spacedriveapp/spacedrive#148] [General]
Hi, I got redirected to here after opening a pull request saying that we might change "atop" (that is a 100% good english word but sounds strange for non-native people) to "above", which is more familiar for everyone.

This would change the "What is a VDFS" section in the README file.

Thank you for your time.

**Comment by utkubakir** [comment]
https://github.com/spacedriveapp/spacedrive/discussions/142
